flow/layers/display_list_layer.cc

```diff
@@ -115,45 +58,14 @@ void DisplayListLayer::Preroll(PrerollContext* context,
   CheckForChildLayerBelow(context);
 #endif
 
-  //   if (auto* cache = context->raster_cache) {
-  //     TRACE_EVENT0("flutter", "DisplayListLayer::RasterCache (Preroll)");
-
-  //     SkMatrix ctm = matrix;
-  //     ctm.preTranslate(offset_.x(), offset_.y());
-  // #ifndef SUPPORT_FRACTIONAL_TRANSLATION
-  //     ctm = RasterCache::GetIntegralTransCTM(ctm);
-  // #endif
-  //     cache->Prepare(context->gr_context, sk_picture, ctm,
-  //                    context->dst_color_space, is_complex_, will_change_);
-  //   }
-
-  // FML_LOG(ERROR) << "display list cull rect is ["
-  //   << cull_rect_.left() << ", "
-  //   << cull_rect_.top() << ", "
-  //   << cull_rect_.right() << ", "
-  //   << cull_rect_.bottom() << "]";
-  // FML_LOG(ERROR) << "display list draw rect is ["
-  //   << draw_rect_.left() << ", "
-  //   << draw_rect_.top() << ", "
-  //   << draw_rect_.right() << ", "
-  //   << draw_rect_.bottom() << "]";
+  // TODO(flar): implement DisplayList raster caching
```

Comment:
nit: please link an issue here explaining what needs to be done, perhaps with a link to the commit(s) containing the start of the implementation.

Reply:
I will want to file issues eventually, but I feel odd filing an issue for a piece of code that isn't merged yet and this code may have a few more weeks before it can be merged.